react/features/mobile/navigation/components/conference/components/ConferenceNavigationContainer.js

```diff
@@ -67,7 +67,6 @@ const ConferenceNavigationContainer = () => {
             ref = { conferenceNavigationRef }
             theme = { navigationContainerTheme }>
             <ConferenceStack.Navigator
-                initialRouteName = { screen.conference.main }
```

Comment:
If we don't specify one, how are we sure we are going to the right place? Which is the first now?

Reply:
I think this is useful only for the main root navigator. Other navigators are screens inside itself.